README.md

```diff
@@ -23,11 +23,11 @@ enhancements are welcome.
 Running the doc server locally is easy:
 
 ```
-$ git clone git://github.com/domcleal/puppetmodule.info
-$ cd puppetmodule.info
-$ bundle install
-$ rake gems:update
-$ rackup config.ru
+git clone git://github.com/voxpupuli/puppetmodule.info
+cd puppetmodule.info
+bundle install --path .vendor --jobs $(nproc)
```

Comment:
`--path` is obsolete and we should use `bundle config path .vendor` (from the top of my head).

Reply:
updated it. it needs to have `--local` if we want to only set the option for the local project.